assessment.md

```diff
@@ -100,4 +100,30 @@ Project Management is Covered by Team 2
 ## Revision of Guides
 
 * Policies for revising the assessment form
+  * 😞 = Revision on unassigned and assigned sections of the assessment form
+      are made without approval, a commit message, and a Pull Request
+  * 😕 = Revision on assigned sections of the assessment form are made with
+      approval. Revisions lack an adequate commit message and a Pull Request
+  * 😐 = Revision on assigned sections of the assessment form are made with
+      approval, an adequate commit message, and a Pull request that tags a TD or
```

Comment:
I think you mean technical leader here. There is also no hyphen in "project manager"